Application/EdFi.Ods.Api/Controllers/IdentitiesController.cs

```diff
@@ -252,7 +253,7 @@ public async Task<IActionResult> Result([FromQuery(Name = "id")] string searchTo
                     case IdentityStatusCode.Success:
                         return Ok(result.Data);
                     case IdentityStatusCode.Incomplete:
-                        Response.Headers["Location"] = Request.GetDisplayUrl();
+                        Response.GetTypedHeaders().Location = new Uri(Request.GetDisplayUrl());
```

Comment:
reason for wrapping in a new Uri?

Reply:
Because Location is of type Uri and GetDisplayUrl() returns string